### PR TITLE
thuang-update-alertaccordion

### PR DESCRIPTION
**Pull request by tihuan**
### Description

Added accordion!

demo

PTAL thank you!!

#### Issue
[ch<fill_in_issue_number>](https://app.clubhouse.io/genepi/story/<fill_in_issue_number>)

### Test plan

Write how your changes are tested, or give a convincing reason why they can't be tested automatically.

### CODE REVIEW

**Review by phoenixAja — APPROVED** [review]
cool this looks good! How would we trigger the accordion?

**Comment by tihuan** [comment]
The arrow to the right of the alert!

File: src/frontend/src/components/AlertAccordion/index.tsx [hunk 1]
```diff
@@ -16,10 +18,24 @@ export default function AlertAccordion({
...
+      className={className}
+      severity={severity}
+      action={
+        <IconButton aria-label="expand" color="inherit" onClick={handleClick}>
+          <ExpandMore fontSize="inherit" />
+        </IconButton>
+      }
+    >
+      {title && <AlertTitle>{title}</AlertTitle>}
+      {isShown && message}
```
Comment (phoenixAja) [inline src/frontend/src/components/AlertAccordion/index.tsx:38]: would message be an array of strings if we want to trigger the accordion?

**Comment by phoenixAja** [comment]
@tihuan sorry i meant if we wanted the table to show up with the different error messages not the accordion action

File: src/frontend/src/components/AlertAccordion/index.tsx (hunk 1 above)
Comment (phoenixAja) [reply src/frontend/src/components/AlertAccordion/index.tsx:38]: sorry again meant error states table

Comment (phoenixAja) [reply src/frontend/src/components/AlertAccordion/index.tsx:38]: I can add that in too if this PR was for the accordion only! I was getting ahead of myself i guess lol

**Comment by tihuan** [comment]
Screen Shot 2021-06-10 at 3 55 49 PM

Worked with Phoenix offline for a proof of concept!

**Pull request merged**